tests/helpers/socialShare-test.js

```diff
@@ -35,4 +42,21 @@ describe('socialShare helper', () => {
             );
         });
     });
+    describe('getSocialShareFn - no hash', () => {
+        const testUrl = encodeURIComponent(getBaseUrlNoHash('spending_exploder'));
```

Comment:
What is the `spending_exploder`? @kbard please advise.

Reply:
🤣

Reply:
All y'all are gonna get me in so much trouble.